components/mixins/experience-event/experienceevent-advertising-stitch.example.1.json

```diff
@@ -0,0 +1,5 @@
+{
+  "xdm:advertisingStitch": {
```

Comment:
This does not match the definition of advertistingStitch. Because you have that field defined as a Map this would need to look something like:
"xdm:advertistingStitch": {
     "some_key_here": {"xdm:id": "1234"}
 }